.pre-commit-config.yaml

```diff
@@ -25,6 +25,6 @@
     sha: e4693a4c282b4fc878eda172a929f7a6508e7d16
     hooks:
       -   id: go-fmt
-          files: (.*\.go)
+          files: \.go$
```

Comment:
We should remove these file patterns because it's already defined in repo `https://github.com/dnephin/pre-commit-golang`